apisix/runner/server/server.py

```diff
@@ -82,6 +84,7 @@ def __init__(self, config: NewServerConfig):
             os.remove(self.fd)
         self.sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
         self.sock.bind(self.fd)
+        os.chown(self.fd, getpwnam('nobody').pw_uid, grp.getgrnam('nobody').gr_gid)
```

Comment:
Another user can be used by configuring the `user` directive: https://nginx.org/en/docs/ngx_core_module.html#user

Reply:
Is there a way I could identify it programmatically (with an ENV var)? Is there a reliable way to find APISIX conf file?

Reply:
> Is there a way I could identify it programmatically (with an ENV var)? Is there a reliable way to find APISIX conf file?

It seems that this will make the matter more complicated. 🤔

Reply:
Or we could add it the the plugin's conf file (and use `nobody as default`)

Reply:
@spacewander : I fixed the tests and added the possibility to override default `socket.owner` from `nobody` => `${whomever you want}`